src/mini.lalrpop

```diff
@@ -159,6 +159,7 @@ Type: Type = {
 	<i: "impure"?> "func" <a: CommaedTypesInParens> <r: ("->" <Type>)?> => Type::Func(i.is_some(), a, Box::new(r.unwrap_or(Type::Void))),
 	"any" => Type::Any,
 	"option<" <t1:Type> ">" => Type::Option(Box::new(t1)),
+	"every" => Type::Every,
```

Comment:
The syntax here will be mysterious to many people who are reading Mini code.  It's true in a formal sense that a program that returns type `every` cannot have any returns, but that requires understanding what the type `every` (which never occurs in a program) means, what its implication is, and that you declare a return type in order to declare that the function never returns.  

Can we add a more straightforward syntax as an alternative?  That could just be syntactic sugar for the `-> every` construct.